varia/sass/full-site-editing/_editor.scss

```diff
@@ -41,6 +41,12 @@
 	}
 }
 
+.post-content__block {
+	margin-bottom: 160px;
+	margin-top: 36px;
+	z-index: 20; // Minimum to appear above the Template block overlay
```

Comment:
🙈 Should we add a sass variable then over the magic number?

Reply:
I don't think it will be of such big help, though...
That magic number is to surpass the [Gutenberg's z-index value for wide- and full-aligned blocks](https://github.com/WordPress/gutenberg/blob/f198997e2c8e377423beb230ce5283914076d396/packages/block-editor/src/components/block-list/style.scss#L496) ([here the value](https://github.com/WordPress/gutenberg/blob/f198997e2c8e377423beb230ce5283914076d396/assets/stylesheets/_z-index.scss#L8)), like the Template.

I'll go ahead and clarify it better in the comment though! 👍